composer.json

```diff
@@ -17,7 +17,7 @@
         "php": ">=5.3",
         "react/event-loop": "0.3.*|0.4.*",
         "react/socket-client": "^0.5",
-        "react/socket": "0.3.*|0.4.*",
+        "react/socket": ">=0.4.5",
```

Comment:
The upcoming v0.5.0 release will introduce BC breaks, so you should probably stick to caret notation.